### ISSUE TITLE
Draft proposal

### ISSUE THREAD

**Issue by YueJiangMDSV** [UBC-MDS/DSCI-532_group-211_dashboards#7]
Hi guys, 

Please take a look at the proposal. Let's discuss where we should improve. Thanks.

Alex

**Comment by rita-ni** [comment]
Hi Alex,

Good job on the proposal!

As we dicussed today, we need to adjust the proposal a little bit to match better with the functions of our dashboard, looking forward to see your revised version. A minor suggestion to your first draft: it may be easier to read if you can split the large chunk of text into paragraphs. Thanks! - Rita

**Comment by arunmarria** [comment]
@YueJiangMDSV, Good job!

I really liked the explanation of the dataset!

Here are a couple of suggestions for other two sections which I think may make the overall proposal better!

1. The proposal  as of now mainly focuses on long term investment but as user can change year range it might be worth mentioning about  both **short term and long term** investment return/values. 
2. You did a great job on use case scenario in last section but I think it will be better if we **explicitly state research questions** first (probably in bullet points) and then continue with the use case. If you feel that we might go overlimit in terms of word count we can reduce use case section. 

What do you guys think? @YueJiangMDSV @rita-ni

**Comment by YueJiangMDSV** [comment]
Thanks @rita-ni @arunmarria !

These are definitely useful suggestions! I'll incorporate what we discussed today into the proposal. 

I think it is a good idea to state the research questions explicitly. I'll update the proposal to reflect that.

**Comment by arunmarria** [comment]
Also @rita-ni I think it would be good idea to have link in readme for proposal. Please do that when you upload final version!

**Comment by rita-ni** [comment]
> Also @rita-ni I think it would be good idea to have link in readme for proposal. Please do that when you upload final version!

Thank you Arun, this is great suggestion, I will add the link to the proposal, as well as our research question to README after Alex revised the proposal.

**Comment by rita-ni** [comment]
@YueJiangMDSV 

Hey Alex, since we used some calculation to come up with the new column I think I may be helpful to use some equations to display them if possible, I created a pull request for you to review the equations I added, if you think they looks good, feel free to merge the change. 

I used this website to create the images: https://www.codecogs.com/latex/eqneditor.php.

Thank you for all the good work!
